docs/_static/nise_usage.inc

```diff
@@ -0,0 +1,13 @@
+Usage
+------
+
+The following is a basic example of using the **nise** module to open a session to a Switch Executive Virtual Device and connect a routegroup.
+
+.. code-block:: python
+
+    import nise
+    with nise.Session('SwitchExecutiveExample') as session:
```

Comment:
Is "'SwitchExecutiveExample'" what we typically use in NISE examples? (LabVIEW, etc)

Reply:
That is the default virtual device automatically added to the system by default when you install Switch Executive.